src/results/ResultsContainer.tsx

```diff
@@ -306,8 +309,21 @@ export class Results extends Component<ResultsProps, ResultsContainerState> {
                 Align
               </button>
               <button type="button" className="button tertiary">
-                <DownloadIcon />
-                Download
+                <Link
+                  to={{
+                    pathname: '/download',
+                    state: {
+                      query,
```

Comment:
As a I think most of these parameters are present in the store and `DownloadContainer` is connected to it, I wonder if it would be nicer to get these from the store directly instead of passing them as props?

Reply:
Yes, no point passing things in the link's state if it is available elsewhere. I'll take these out.

Reply:
Looking at each of the bits of data passed in the `Link`:

Taken from the URL within the `ResultsContainer` and then updated in the store:
* `query`

Taken from the URL within the `ResultsContainer` but not updated in the store:
* `selectedFacets`
* `sortColumn`
* `sortDirection`

Only exists as local state to `ResultsContainer`:
* `selectedEntries`

It looks like the only thing I could take is the `query` - would you prefer this to be taken from the store rather than passed via the `Link`?

Reply:
No that's fine. Had forgotten that the main state came from the url